src/App.js

```diff
@@ -1,12 +1,38 @@
 
+import React, { useState } from "react"
+import Header from "./components/Header"
+import Footer from "./components/Footer"
+import CatEdit from "./pages/CatEdit"
+import CatIndex from "./pages/CatIndex"
+import CatNew from "./pages/CatNew"
+import CatShow from "./pages/CatShow"
+import Home from "./pages/Home"
+import NotFound from "./pages/NotFound"
 import './App.css';
+import './index.css';
+import mockCats from "./mockCats"
+import { Routes, Route } from "react-router-dom"
 
-function App() {
+
+const App = () => {
+  const [cats, setCats] = useState(mockCats)
   return (
- <>
- <h1>Hello World</h1>
+ <> <div className = "header">
+ < Header /> 
+ </div>
+ <Routes>
+    <Route path="/" element = {<Home/>} />
+    <Route path="/catindex" element = {< CatIndex/>} />
+    <Route path="/catshow" element = {< CatShow/>} />
+    <Route path="/catnew" element = {< CatNew/>} />
+    <Route path="/catedit" element = {<CatEdit/>} />
+    <Route path="*" element = {< NotFound/>} />
+ </Routes> 
+  <div className = "footer">
+ < Footer /> 
+ </div>
  </>
   );
-}
+};
```

Comment:
Hey - you don't have to use the divs for header and footers in App.js.  Remember, these are components, so any structure you'd like those things to have will live in their component files (i.e. Header.js/Footer.js).